doc/ukernel/operations/transform.md

```diff
@@ -34,7 +44,9 @@ No attribute is supported for transform ukernel.
 
 ## Implementation limitations
 
-- Destination leading dimension only supported values are: 16, 32, 48, or 64.
+- Destination leading dimension, or `out_ld`, only supported values are: `16`,
+  `32`, `48`, or `64`. This is the implementation limitation, there's no
+  effecient kernels supported for other leading dimension values.
```

Comment:
```suggestion
  efficient kernels supported for other leading dimension values.
```

Reply:
Could you please clarify if there are no efficient kernels or kernels at all for other leading dimensions?

Reply:
Thanks for the suggestions, incorporated.

From the end-user perspective it should be no difference whether there's no kernels or they are not efficient at all. In theory we can plug simple_reorder in, but it would be no point in doing that. It's just a comment explaining where the limitation is coming from.